src/components-examples/journey-maps/angular/journey-maps-routes-zones/journey-maps-routes-zones-example.ts

```diff
@@ -0,0 +1,247 @@
+import { AsyncPipe, DecimalPipe, TitleCasePipe } from '@angular/common';
+import { ChangeDetectorRef, Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { ReactiveFormsModule, UntypedFormBuilder, UntypedFormGroup } from '@angular/forms';
+import { SbbButtonModule } from '@sbb-esta/angular/button';
+import { SbbCheckboxModule } from '@sbb-esta/angular/checkbox';
+import { SbbOptionModule } from '@sbb-esta/angular/core';
+import { SbbFormFieldModule } from '@sbb-esta/angular/form-field';
+import { SbbIconModule } from '@sbb-esta/angular/icon';
+import { SbbInputModule } from '@sbb-esta/angular/input';
+import { SbbNotificationModule } from '@sbb-esta/angular/notification';
+import { SbbRadioButtonModule } from '@sbb-esta/angular/radio-button';
+import { SbbSelectModule } from '@sbb-esta/angular/select';
+import {
+  SbbJourneyMaps,
+  SbbJourneyMapsModule,
+  SbbJourneyMapsRoutingOptions,
+  SbbViewportDimensions,
+  SbbZoomLevels,
+} from '@sbb-esta/journey-maps';
+import { Feature, FeatureCollection, Polygon, Position } from 'geojson';
+import { LngLatBounds, LngLatBoundsLike, LngLatLike } from 'maplibre-gl';
+import { BehaviorSubject, Subject } from 'rxjs';
+import { filter, map, take, takeUntil } from 'rxjs/operators';
+
+import {
+  JOURNEY_MAPS_DEFAULT_ROUTING_OPTIONS,
+  JOURNEY_MAPS_DEFAULT_ZONE_OPTIONS,
+} from '../shared/config';
+
+declare global {
+  interface Window {
+    JM_API_KEY: string;
+  }
+}
+
+/**
+ * @title Journey Maps - SBB Map Routes & Zones
+ * @includeExtraFiles ../shared/config.ts,../shared/markers.ts,../shared/journey/be-sh.ts,../shared/journey/zh-be_wyleregg.ts,../shared/journey/zh-sh_waldfriedhof.ts,../shared/routes/biel-lyss.ts,../shared/routes/bn-ls.ts,../shared/transfer/bern-indoor.ts,../shared/transfer/geneve-indoor.ts,../shared/transfer/luzern4-j.ts,../shared/transfer/zurich-indoor.ts,../shared/zone/bern-burgdorf.ts,../shared/zone/bs-bl.ts
```

Comment:
This annotation @includeExtraFiles with routes and zones files, you need only in the Stackblitz example “SBB Map Routes & Zones”.
My suggestion: inside the shared/config.ts file, keep only the imports for zone routes only.
The remaining Consts values, copy directly into the concrete example:
- style_ids
- rail_colors
- ch_bounds
- poi_categories